tests/test_commands_windows.py

```diff
@@ -162,6 +162,17 @@ def test_flow():
     assert len(glob.glob(f"{temp_dir}\\prefect-*.ps1")) == 0
 
 
+def test_shell_run_command_throw_exception_on_nonzero_exit_code():
+    @flow
+    def test_flow():
+        return shell_run_command(
+            command="ping ???", shell="powershell"  # ping ??? returns exit code 1
+        )
+
+    with pytest.raises(RuntimeError):
```

Comment:
Can you also use the `match` argument of `pytest.raises` to make sure that the exception message contains the expected error code?